src/OrchardCore/Orchard.Localization.Abstractions/ILocalizationFileLocationProvider.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Orchard.Localization.Abstractions
+{
+    public interface ILocalizationFileLocationProvider
```

Comment:
I remember I asked you to add this interface. But I think I added another comment when I realized this interface already existing in ASP.NET Core and we should use the same one.